listener.go

```diff
@@ -314,15 +319,16 @@ func (l *Listener) handleMsg(buf []byte, oob *ipv4.ControlMessage, _peer net.Add
 	}
 
 	ll.Infof(
-		"%s to %s on %s with %s, lease %s, hostname %s.%s, tftp %s:%s",
+		"%s to %s on %s with %v, lease %s, hostname %s.%s, tftp %s:%s",
 		resp.MessageType(),
 		peer.IP,
 		ifi.Name,
 		pickedIP,
 		*flagLeaseTime,
-		hostname,
-		domainname,
-		tftp,
+		*options.Hostname,
```

Comment:
Updated so it logs out what is actually sent in the reply (which my previous PR broke)